test/integration/test_resolvers.py

```diff
@@ -0,0 +1,24 @@
+"""Integration tests for the conda dependency resolution."""
+
+import os
+from base import integration_util
+from base.api import ApiTestCase
+
+GNUPLOT = {u'version': u'4.6', u'type': u'package', u'name': u'gnuplot'}
+
+class CondaResolutionIntegrationTestCase(integration_util.IntegrationTestCase, ApiTestCase):
```

Comment:
@jmchilton this test fails with 
```
ERROR: test suite for <class 'integration.test_resolvers.CondaResolutionIntegrationTestCase'>
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/marius/src/galaxy/.venv/lib/python2.7/site-packages/nose/suite.py", line 209, in run
    self.setUp()
  File "/Users/marius/src/galaxy/.venv/lib/python2.7/site-packages/nose/suite.py", line 292, in setUp
    self.setupContext(ancestor)
  File "/Users/marius/src/galaxy/.venv/lib/python2.7/site-packages/nose/suite.py", line 315, in setupContext
    try_run(context, names)
  File "/Users/marius/src/galaxy/.venv/lib/python2.7/site-packages/nose/util.py", line 471, in try_run
    return func()
  File "/Users/marius/src/galaxy/test/base/integration_util.py", line 23, in setUpClass
    cls._test_driver.setup(config_object=cls)
  File "/Users/marius/src/galaxy/test/base/driver_util.py", line 641, in setup
    galaxy_config,
  File "/Users/marius/src/galaxy/test/base/driver_util.py", line 521, in launch_server
    static_enabled=True
  File "/Users/marius/src/galaxy/lib/galaxy/webapps/galaxy/buildapp.py", line 47, in app_factory
    return paste_app_factory( global_conf, **kwargs )
  File "/Users/marius/src/galaxy/lib/galaxy/webapps/galaxy/buildapp.py", line 84, in paste_app_factory
    webapp.add_ui_controllers( 'galaxy.webapps.galaxy.controllers', app )
  File "/Users/marius/src/galaxy/lib/galaxy/web/framework/webapp.py", line 115, in add_ui_controllers
    module = import_module( module_name )
  File "/usr/local/Cellar/python/2.7.10_2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/Users/marius/src/galaxy/lib/galaxy/webapps/galaxy/controllers/admin_toolshed.py", line 32, in <module>
    from tool_shed.util import workflow_util
  File "/Users/marius/src/galaxy/lib/tool_shed/util/workflow_util.py", line 14, in <module>
    from galaxy.workflow.render import WorkflowCanvas
  File "/Users/marius/src/galaxy/lib/galaxy/workflow/render.py", line 1, in <module>
    import svgwrite
  File "/Users/marius/src/galaxy/.venv/lib/python2.7/site-packages/svgwrite/__init__.py", line 51, in <module>
    from svgwrite.drawing import Drawing
  File "/Users/marius/src/galaxy/.venv/lib/python2.7/site-packages/svgwrite/drawing.py", line 29, in <module>
    from svgwrite.container import SVG, Defs
  File "/Users/marius/src/galaxy/.venv/lib/python2.7/site-packages/svgwrite/container.py", line 27, in <module>
    from svgwrite.base import BaseElement
  File "/Users/marius/src/galaxy/.venv/lib/python2.7/site-packages/svgwrite/base.py", line 16, in <module>
    from svgwrite.params import Parameter
  File "/Users/marius/src/galaxy/.venv/lib/python2.7/site-packages/svgwrite/params.py", line 9, in <module>
    from svgwrite.validator2 import get_validator
  File "/Users/marius/src/galaxy/.venv/lib/python2.7/site-packages/svgwrite/validator2.py", line 9, in <module>
    from svgwrite.data import full11
  File "/Users/marius/src/galaxy/.venv/lib/python2.7/site-packages/svgwrite/data/full11.py", line 11, in <module>
    from svgwrite.data.typechecker import Full11TypeChecker as TypeChecker
  File "/Users/marius/src/galaxy/.venv/lib/python2.7/site-packages/svgwrite/data/typechecker.py", line 13, in <module>
    from svgwrite.data.svgparser import is_valid_transferlist, is_valid_pathdata, is_valid_animation_timing
  File "/Users/marius/src/galaxy/.venv/lib/python2.7/site-packages/svgwrite/data/svgparser.py", line 26, in <module>
    comma = Literal(',') * (0, 1)  # zero or one ','
TypeError: can't multiply sequence by non-int of type 'Literal'
```
Uncommenting the offending lines in svgwrite makes the tests pass ... any idea why this happens here, but not when running galaxy (or the regular API tests) ?